src/Main.java

```diff
@@ -0,0 +1,172 @@
+import java.io.*;
```

Comment:
Avoid multiple imports.
Use explicit imports:
```suggestion
import java.io.File
import java.io.BufferedWriter;
import java.io.FileWriter;
```
and so on